handlers/main.yml

```diff
@@ -1,15 +1,7 @@
 # file: postgresql/handlers/main.yml
 
-  - name: restart postgresql with service
-    service:
-      name: "{{ postgresql_service_name }}"
-      state: restarted
-      enabled: yes
+- name: reload postgresql
+  include_tasks: "{{ role_path }}/handlers/reload.yml"
```

Comment:
you should instead of using this, add this to the handlers/main.yml already. This gets out of best practices and making things more complicated. Just use the same file.